### ISSUE TITLE
Unwanted TLS package captured

### ISSUE THREAD

**Issue by slomo** [buger/goreplay#892]
When listing for traffic with `--input-raw <port>` gor will also capture responses to outgoing https connections (probably all tls traffic), where the destination port is the same as the capture port.

My original issue occurred on internal services all running on 8080 (some with TLS, some without). To replicate here is the same setup with using 443 and google.

Running gor with:
```
./gor  --input-raw :443 --output-stdout --http-allow-url /exampelUrl
```

Making requests like:
```
curl https://google.de
```

Following data is logged:
```
2021/01/25 09:39:38 [PPID 31862 and PID 9100] Version:1.2.0
1 cf9825d7165d6ce711d88782 1611563981259754731 101399191
[[ lots of binary ]]
                     	google.de
                                 


3t
0. hhttp/1.1


[[ even more binary ]]
```

I observed the behavior in 1.2 and 1.3-RC1, not sure if it is a bug or expected. At least 1.0 behaves differently.

**Comment by buger** [comment]
I do not think that capturing TLS like this is a good idea. You will not be able to replay it. 
The best bet here is to make GoReplay act as a proxy so you will send traffic though it.

**Comment by slomo** [comment]
I see maybe that was unclear from my original response: I don't want to capture TLS traffic. I want only the inbound HTTP traffic but I also get the replys to outbound HTTPS connections.

My setup is I have service A on host 1 (port 8080 plain HTTP), that I want to replicate using gore to some downstream environment) and I have service B in host 2 (port 8080 HTTPS), gets requests from service B for certain tasks.

Now I try to capture traffic on host 1 using `gor --raw-input :8080` and I get those binary fragments from the connection between host 1/service a and host 2/service b.

This also happens for plain connections (I only didn't see it due to my url filter):

```
$ curl google.de
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="http://www.google.de/">here</A>.
</BODY></HTML>
```

```
$ ./gor  --input-raw :80 --output-stdout
2021/01/26 07:36:14 [PPID 2424 and PID 3936] Version:1.2.0
1 e951b087165db4c081a78eb8 1611642980550642838 19825186
GET / HTTP/1.1
Host: google.de
User-Agent: curl/7.58.0
Accept: */*
```

I have accepted that this is expected behavior. Maybe it could just be documented somewhere, to avoid people like me from opening issues ;). All the examples on the Readme and Getting Started omit this part, and also it used to work differently in 1.0.0

**Comment by buger** [comment]
Ah got it now. 
I think it can be treated as an issue. 
It should capture only inbound traffic here.
Will re-open.
